Extensions/mutualchecker.js

```diff
@@ -49,12 +49,12 @@ XKit.extensions.mutualchecker = new Object({
 		$(".follower").each(function() {
 			if (following || !$(this).find(".follow_button").length) {
 				var $name_div = $(this).find(".name-link");
-				var url = $name_div.text();
-				XKit.interface.is_following(url, XKit.extensions.mutualchecker.preferences.main_blog.value).then(function(follow) {
-					if (follow) {
-						XKit.extensions.mutualchecker.add_label($name_div, url);
-					}
-				});
+				var fake_obj = {
```

Comment:
what's `fake_obj` ? could you use a more descriptive name than this or leave an inline comment?